bellatrix-1/README.md

```diff
@@ -0,0 +1,269 @@
+## Software requirements
+
+Stargaze has releases for Linux and MacOS [here](https://github.com/public-awesome/stargaze/releases/tag/v0.6.0).
+
+- [Ubuntu Setup Guide](./ubuntu.md)
+- Latest version : [v0.6.0](https://github.com/public-awesome/stargaze/releases/tag/v0.6.0)
+
+### Install Stargaze
+
+You can install Stargaze by downloading the binary (easiest), or compiling from source.
+
+#### Option 1: Download binary
+
+1. Download the binary for your platform: [releases](https://github.com/public-awesome/stargaze/releases/tag/v0.6.0).
+2. Copy it to a location in your PATH, i.e: `/usr/local/bin` or `$HOME/bin`.
+
+i.e:
+```sh
+> wget https://github.com/public-awesome/stargaze/releases/download/v0.6.0/stargaze_0.3.2_linux_arm64.tar.gz
+> sudo tar -C /usr/local/bin -zxvf stargaze_0.3.2_linux_arm64.tar.gz
+```
+
+#### Option 2: Build from source
+
+Requires [Go version v1.15+](https://golang.org/doc/install).
+
+```sh
+> mkdir -p $GOPATH/src/github.com/public-awesome
+> cd $GOPATH/src/github.com/public-awesome
+> git clone https://github.com/public-awesome/stargaze && cd stargaze
+> git checkout v0.6.0
+> FAUCET_ENABLED=true make install
+```
+
+#### Verify installation
+
+To verify if the installation was successful, execute the following command:
+
+```sh
+> starsd version --long
+```
+
+It will display the version of starsd currently installed:
+
+```sh
+name: stargaze
+server_name: starsd
+version: 0.6.0
+commit: a152fc6aab2f3259d24e071f58855cf445236b55
+build_tags: netgo,faucet
+go: go version go1.15.8 linux/amd64
+```
+
+NOTE: Make sure `build_tags` includes "faucet", which is required for testnet.
+
+## Setup validator node
+
+If you are looking to join the testnet post genesis time (_Nov 03 2020 1600 UTC_), skip to [Create Testnet Validator](#create-testnet-validator)
+
+Below are the instructions to generate & submit your genesis transaction
+
+### Generate genesis transaction (gentx)
+
+1. Initialize the Stargaze directories and create the local genesis file with the correct
+   chain-id
+
+   ```sh
+   > starsd init <moniker-name> --chain-id=bellatrix-1
+   ```
+
+2. Create a local key pair
+
+   ```sh
+   > starsd keys add <key-name>
+   ```
+
+3. Add your account to your local genesis file with a given amount and the key you
+   just created. Use only `1000000000ustarx`, other amounts will be ignored. STARX is testnet STAR.
+
+   ```sh
+   > starsd add-genesis-account $(starsd keys show <key-name> -a) 1000000000ustarx
+   ```
+
+4. Create the gentx
+
+   ```sh
+   > starsd gentx <key-name> --amount=900000000ustarx --chain-id=bellatrix-1
+   ```
+
+   If all goes well, you will see a message similar to the following:
+
+    ```sh
+    Genesis transaction written to "/home/user/.starsd/config/gentx/gentx-******.json"
+    ```
+
+### Submit genesis transaction
+
+> NOTE: To prevent malicious validators, and to ensure a fair and decentralized launch, the following rules will be enforced:
+> 1. Github accounts must be at least a year old
```

Comment:
"Github accounts must be at least a year old and have history; accounts with little activity may not be accepted."